third_party/sphinx/sphinx/ext/autosummary/templates/autosummary/class.rst

```diff
@@ -0,0 +1,33 @@
+{{ fullname | escape | underline}}
+
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
```

Comment:
If we are customizing, then `:inherited-members:` should go here?

Reply:
Yes, we are customizing slightly (the toctree so that sub-pages are generated). I'll add a comment.

Somehow that doesn't seem necessary for autosummary. I see the inherited members at https://storage.googleapis.com/tswast-docs-staging/google-cloud-python/pr-5340/bigquery/generated/google.cloud.bigquery.job.LoadJob.html#google.cloud.bigquery.job.LoadJob